### ISSUE TITLE
LAPACK has a C interface now.

### ISSUE THREAD

**Issue by Skylion007** [nagadomi/animeface-2009#5]
There have been numerous [improvements](http://www.netlib.org/lapack/improvement.html) to LAPACK since 2009 including a C interface. Having the library link against the newer, optimized versions of LAPACK would not only speed up the code, but make it easier to contribute to.

**Comment by nagadomi** [comment]
I agree with that. It would be better to be able to change BLAS / LAPACK to OpenBLAS or something. But I don't have the motivation to improve this codebase :sob: (`nvxs` is a very old fork of https://github.com/nagadomi/nv and currently I don't use that library )
As far as I can remember, `./nvxs/CLAPACK` is Fortran interface version, changing integer type (long?) to work on Windows.

**Comment by Skylion007** [comment]
So I was digging through the code and to my great surprise their appears to be code for GPU support as well? Does it actually work? If so, would you consider modifying the build.sh to include an easy flag for it?

**Comment by nagadomi** [comment]
I used this code with CUDA, but in 2008 I did not release that code because GPGPU was not popular.
I put that code in [appendix/cuda](https://github.com/nagadomi/animeface-2009/tree/master/appendix/cuda). However, I have not confirmed that the code still works.
